manifests/init.pp

```diff
@@ -183,13 +183,12 @@
   String $oauth_key = 'candlepin',
   String $oauth_secret = 'candlepin',
   Boolean $env_filtering_enabled = true,
-  Stdlib::Absolutepath $keystore_file = '/etc/candlepin/certs/keystore',
-  Optional[String] $keystore_password = undef,
-  String $keystore_type = 'PKCS12',
-  Stdlib::Absolutepath $truststore_file = '/etc/candlepin/certs/truststore',
-  Optional[String] $truststore_password = undef,
-  Stdlib::Absolutepath $ca_key = '/etc/candlepin/certs/candlepin-ca.key',
-  Stdlib::Absolutepath $ca_cert = '/etc/candlepin/certs/candlepin-ca.crt',
+  Optional[String] $keystore_password = $candlepin::params::keystore_password,
+  Optional[String] $truststore_password = $candlepin::params::truststore_password,
+  Stdlib::Absolutepath $ca_key = undef,
+  Stdlib::Absolutepath $ca_cert = undef,
+  Stdlib::Absolutepath $tomcat_ssl_cert = undef,
+  Stdlib::Absolutepath $tomcat_ssl_key = undef,
```

Comment:
Fun bikeshed question, should we be starting to use `tls` instead of `ssl` or some other term all together to avoid that distinction?

Reply:
We're not very consistent with ssl/tls. Right now we mostly use ssl but tls would be more correct. Probably a whole separte discussion.